src/App.tsx

```diff
@@ -15,48 +17,110 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
+enum SortType {
+  alphabet = 'Sort alphabetically',
+  length = 'Sort by length',
+}
+
+interface SortParams {
+  sortField: string,
+  isReversed: boolean,
+}
+
+function getPreparedGoods(
+  goods: string[],
+  { sortField, isReversed }: SortParams,
+) {
+  let preparedGoods = [...goods];
+
+  if (sortField) {
+    preparedGoods = preparedGoods.sort((good1, good2) => {
+      switch (sortField) {
+        case SortType.alphabet:
+          return good1.localeCompare(good2);
+
+        case SortType.length:
+          return good1.length - good2.length;
+
+        default:
+          return 0;
+      }
+    });
+  }
+
+  if (isReversed) {
+    return preparedGoods.reverse();
+  }
+
+  return preparedGoods;
+}
+
 export const App: React.FC = () => {
+  const [sortField, setSortField] = useState('');
```

Comment:
```suggestion
  const [sortField, setSortField] = useState(SortType.default);
```
add this as the default option of the enum